src/Rules/PropertyConditions/BoundingRectangle.cs

```diff
@@ -74,5 +75,11 @@ private static bool ElementCompletelyObscuresContainer(IA11yElement e)
 
             return e.BoundingRectangle.CompletelyObscures(container.BoundingRectangle);
         }
+        private static bool ElementItemStatusContainsTelerikSparklineContext(IA11yElement e)
+        {
+            if (e == null) throw new ArgumentNullException(nameof(e));
+
+            return e.Framework == "WPF" && e.ItemStatus?.Contains("<Property Name=\"DataContext\" Value=\"Telerik.Windows.Controls.Sparklines.SparklineColumnDataPoint\" />") == true;
```

Comment:
If we apply the WPF check at the rule, this would become
```suggestion
            return e.ItemStatus?.Contains("<Property Name=\"DataContext\" Value=\"Telerik.Windows.Controls.Sparklines.SparklineColumnDataPoint\" />") == true;
```